### ISSUE TITLE
Error with test files

### ISSUE THREAD

**Issue by patriciatran** [rpetit3/vcf-annotator#11]
Hello,

I have `vcf-annotator.py 0.7` and `Python 3.11.4`, which I installed in a container using:
```
Bootstrap: docker
From: continuumio/miniconda3:23.5.2-0

%post
        conda install -c bioconda -c conda-forge vcf-annotator
```
I was running into some issue with my real data, so I tried running it on the sample data posted in the github repo, but encounter the same type of error. Have you (or others) seen this before or do you have suggestions on ways to proceed?

Thank you.

```
# download data
wget https://github.com/rpetit3/vcf-annotator/raw/refs/heads/master/example-data/example-multi.vcf
wget https://github.com/rpetit3/vcf-annotator/raw/refs/heads/master/example-data/example-multi.gb

# start container:
apptainer shell -e /staging/<username>/apptainer/vcf-annotator.sif 

# run command
Apptainer> vcf-annotator example-multi.vcf example-multi.gb
Traceback (most recent call last):
  File "/opt/conda/bin/vcf-annotator", line 398, in <module>
    annotator.write_vcf(args.output)
  File "/opt/conda/bin/vcf-annotator", line 192, in write_vcf
    self.__vcf.write_vcf(output)
  File "/opt/conda/bin/vcf-annotator", line 357, in write_vcf
    vcf_writer = vcf.Writer(open(output, 'w'), self.reader)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/lib/python3.11/site-packages/vcf/parser.py", line 642, in __init__
    self.writer = csv.writer(stream, delimiter="\t",
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: "quotechar" must be a 1-character string
```

**Comment by rpetit3** [comment]
Hi @patriciatran 

I was able to replicate this and it seems to be related to Python 3.11. Can you try the following?

```
conda install -c conda-forge -c bioconda vcf-annotator 'python<=3.10'
```

**Comment by patriciatran** [comment]
Thanks for your response! I've finally been able to test this. 
I ended up using the method of installing Miniconda on the remote machine for each job, and using conda install there directly (no containers used) and it worked. (I didn't clarify this in my original github issue)

For some reason, specifying the `'python<=3.10'` in the apptainer def file did not work, so I didn't end up using the container method. 

I'll use this solution moving forward for this program:
```
echo "installing miniconda..."
export HOME=$PWD
export PATH
wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh

sh Miniconda3-latest-Linux-x86_64.sh -b -p $PWD/miniconda3
export PATH=$PWD/miniconda3/bin:$PATH

conda install -c conda-forge -c bioconda vcf-annotator 'python<=3.10'
```

thanks
